### ISSUE TITLE
Increase the accuracy of the warning about liquidity providing and loss of funds

### ISSUE THREAD

**Issue by akreider** [AugurProject/turbo#1498]
"Remove liquidity before the winning outcome is known to prevent any loss of funds"

Should be changed to reflect the level of risk involved:
"Remove liquidity before the winning outcome is known to prevent total loss of funds"

One user in the discord reports that they read "any" as an indicator that they wouldn't lose all their funds.

**Comment by JohnDanz** [comment]
@akreider how about this wording:

```
Remove liquidity before the winning outcome is known. Failure to do so can result in a total loss of funds.
```

still convey's the potential risk level while still technically being true if you lose only partial funds

**Comment by akreider** [comment]
That's great!
